sdks/java/container/license_scripts/requirement.txt

```diff
@@ -17,5 +17,6 @@
 ###############################################################################
 
 beautifulsoup4>=4.9.0,<5.0
-pyyaml>=3.12,<6.0.0
+pyyaml>=3.12,<6.0.0; python_version < '3.11'
+pyyaml>=6.0.1,<7; python_version >= '3.11'
```

Comment:
could we simplify this to 
```pyyaml<7``` or even ```pyyaml```?

Reply:
makes sense. Need 6.0.1 (first version supports py3.11)